### ISSUE TITLE
Even Better TOML doesn't respects "publish = false" for Cargo.toml

### ISSUE THREAD

**Issue by boozook** [tamasfe/taplo#440]
### Example toml
```toml
[package]
name = "foo"
version = "0.1.0"
edition = "2021"
publish = false
```

### Problem

Even Better TOML throws errors for entire Cargo.toml like this
```
"Categories" is a required property
"License" is a required property
"Builds" is a required property
```

### Expected result

No errors because of property `publish` is set to `false` and some validations are unnecessary.

- - -

_Probably that is problem of schema, I don't know..._ 🤷🏻‍♂️

**Comment by ia0** [comment]
I am not able to reproduce. I create the following file named `Cargo.toml`:

```toml
[package]
name = "foo"
version = "0.1.0"
edition = "2021"
publish = false
```

And I don't get any errors neither with taplo-lsp, nor taplo-cli.

**Comment by boozook** [comment]
I'll try to isolate it in near future. 
<img width="869" alt="Screenshot 2023-08-08 at 19 27 59" src="https://github.com/tamasfe/taplo/assets/888526/2ac05da0-2362-4827-9116-1459e2268394">

**Comment by ia0** [comment]
Given it looks like you're using VSCode, I suspect this is due to the wrong schema being selected. This was fixed in #426 and made it to 0.19.1 but was reverted in 0.19.2 because of an unrelated crash in #434. You could try to use 0.19.1 and see if it fixes your error (assuming you don't hit the crash first).

**Comment by boozook** [comment]
@ia0 I've tied 0.19.1 and you're right, there's no errors.

**Comment by ia0** [comment]
Thanks for the feedback!

So similarly to https://github.com/tamasfe/taplo/issues/442#issuecomment-1673480147, I would suggest continuing using that version and reporting any crash to #434. Ultimately, 0.19.2 is only a temporary rollback to 0.19.0.

**Comment by boozook** [comment]
And yes, I've got an error :)

**Comment by ia0** [comment]
> And yes, I've got an error :)

Oh no :-( ok then let's follow up on #434. You can subscribe to #434 to be notified when it's fixed and 0.19.3 is released.

**Comment by boozook** [comment]
Thank you. 
I've simply updated back to the latest and set url to cargo.json schema. So works, I suppose... Not yet tested properly.
